lib/components/Avatar.js

```diff
@@ -53,7 +63,7 @@ function Avatar({ size, user, isSquare, status, onClick, ...otherProps }) {
     "nui-avatar--large": isLarge,
     "nui-avatar--xlarge": isXLarge,
     "nui-avatar--round": !isSquare,
-    "hidden": !loaded,
+    hidden: !loaded,
```

Comment:
@amaldinesh7 _a `hidden` should be `"hidden"` in order for prefixer to work. Can we create a demo video and showcase all the conditions.